packages/dvilela/skills/memeooorr_abci/rounds.py

```diff
@@ -184,7 +189,9 @@ def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Event]]:
     required_class_attributes = ()
 
 
-class PostTweetRound(CollectSameUntilThresholdRound):
+class PostTweetRound(
+    CollectSameUntilThresholdRound
+):  # pylint: disable=too-many-return-statements
```

Comment:
@dvilelaf 

disabled a check here , please let me know if it is very critical !